subprojects/packagefiles/sqlpp11/meson.build

```diff
@@ -1,6 +1,7 @@
 project('sqlpp11', 'cpp', version: '0.64', license: 'BSD-2-Clause')
 
-meson.override_find_program('ddl2cpp', files('scripts/ddl2cpp'))
+meson.override_find_program('ddl2cpp', find_program('scripts/ddl2cpp'))
+meson.override_find_program('sqlite2cpp', find_program('scripts/sqlite2cpp.py'))
```

Comment:
This change is based on this comment: https://github.com/mesonbuild/meson/discussions/10978#discussioncomment-11300151